lib/__tests__/standalone-fix.test.mjs

```diff
@@ -39,35 +37,16 @@ it('fixes when enabled in config', async () => {
 	expect(output).toBe('a { color: #fff; }');
 });
 
-it('apply indentation autofix at last', async () => {
```

Comment:
question

If I understand correctly there are no more rules where order matters?
i.e. this was limited to indentation only

Reply:
There are some, e.g. [`property-no-redundant-values`](https://stylelint.io/user-guide/rules/shorthand-property-no-redundant-values) needs to run after [`declaration-block-no-redundant-longhand-properties`](https://stylelint.io/user-guide/rules/declaration-block-no-redundant-longhand-properties), but users less often encounter problems. Many stylistic rules caused `indentation` warnings, so we forced it to run last as it would normally run early on because it starts with an "i".

Reply:
If you can remember please list them here, so that we can refer to the comment if we ever need to document it.

Reply:
I'll jot them down if I remember any more, but can't offhand at the moment.